### PR TITLE
Many comment edits; epsilon-->membership, rephrasings.

### PR DESCRIPTION
**Pull request by benjub**
I find the edits rather straightforward, but there are many of them. Maybe someone could put himself as a reviewer, to clarify who does what ?

### CODE REVIEW

**Review by mazsa — CHANGES_REQUESTED** [review]
@benjub Benoit, under your review https://github.com/metamath/set.mm/pull/2502#pullrequestreview-883776047 I asked you not to do elementhood --> membership replacements, at least not in my Mathbox. I wrote "I´ll explain later, well before these theorems will be in the main set.mm". I know it's been a while, but I'm *still* working on this topic and I promise again to return to it as soon as possible, so I'm still asking that we not do elementhood --> membership , or epsilon --> membership , replacements, again at least not globally, including my Mathbox. I will actively reopen this issue and let you know directly when I do - and it is quite possible that my future proposal will be no different from your current one.

**Comment by benjub** [comment]
> @benjub Benoit, under your review [#2502 (review)](https://github.com/metamath/set.mm/pull/2502#pullrequestreview-883776047) I asked you not to do elementhood --> membership replacements, at least not in my Mathbox. I wrote "I´ll explain later, well before these theorems will be in the main set.mm". I know it's been a while, but I'm _still_ working on this topic and I promise again to return to it as soon as possible, so I'm still asking that we not do elementhood --> membership , or epsilon --> membership , replacements, again at least not globally, including my Mathbox. I will actively reopen this issue and let you know directly when I do - and it is quite possible that my future proposal will be no different from your current one.

Ok, I'll revert the changes in your mathbox.

File: set.mm [hunk 1]
```diff
@@ -50761,14 +50761,13 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
       PVHBWAVOVGVNVFVEFUKULUMUNUOUPURUQUSUTVA $.
   $}
 
-  $( Irreflexivity of the epsilon relation: a class founded by epsilon is not a
-     member of itself.  (Contributed by NM, 18-Apr-1994.)  (Revised by Mario
-     Carneiro, 22-Jun-2015.) $)
+  $( A well-founded class does not belong to itself.  (Contributed by NM,
+     18-Apr-1994.)  (Revised by Mario Carneiro, 22-Jun-2015.) $)
```
Comment (avekens) [inline set.mm:50765]: To understand this comment, the term "well-founded class" must be defined/explained. In my understanding, a class is "well-founded" if the epsilon relation is well-founded over this class (`Fr` is primarily a predicate for a relation ("well-formed relation")). By this , we would get a correlation with the `_E` in the theorem.

File: set.mm [hunk 2]
```diff
@@ -50906,10 +50904,10 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
       UTVAVDUSVFUTVAVDSSUSVFMUTVAVDABCDFEUDUFUGUHUIURUSUTVEVAURUTMDETZVEURCETUT
       VGCEUJDCEUKULABDEUMUNUOVCADUQUP $.
 
-    $( All nonempty (possibly proper) subclasses of ` A ` , which has a
-       well-founded relation ` R ` , have ` R `-minimal elements.  Proposition
-       6.26 of [TakeutiZaring] p. 31.  (Contributed by Scott Fenton,
-       29-Jan-2011.)  (Revised by Mario Carneiro, 24-Jun-2015.) $)
+    $( All nonempty subclasses of ` A ` , which has a well-founded relation
+       ` R ` , have ` R `-minimal elements.  Proposition 6.26 of
```
Comment (avekens) [inline set.mm:50908]: "well-ordered and set-like" instead of "well-founded"?

File: set.mm [hunk 3]
```diff
@@ -55107,8 +55108,10 @@ the restriction (of the relation) to the singleton containing this
 
   ${
     $d x y A $.  $d x y B $.  $d x y C $.
-    $( Subset of the range of a restriction.  (Contributed by NM, 16-Jan-2006.)
-       (Proof shortened by Peter Mazsa, 2-Oct-2022.) $)
+    $( Two ways to express surjectivity of a restricted and corestricted binary
+       relation (intersection of a binary relation with a Cartesian product).
+       (Contributed by NM, 16-Jan-2006.)  (Proof shortened by Peter Mazsa,
+       2-Oct-2022.) $)
```
Comment (avekens) [inline set.mm:55114]: The former "Subset of the range of a restriction. " describes precisely the LHS of the statement. Therefore it should remain in the comment (somehow). The term "corestricted" is not used/defined in set.mm yet.

Comment (avekens) [reply set.mm:55114]: Now I see:  "restricted and corestricted binary relation" means "intersection of a binary relation with a Cartesian product".

File: set.mm [hunk 4]
```diff
@@ -55165,19 +55172,19 @@ the restriction (of the relation) to the singleton containing this
     CUAVPVLUTCUBPUCVLVOABCOUDPVOVJVNBVBBUEUFPUGZVCVKGNGVCVJGVCVJGBDGVBGBUHBUIQU
     JUMQRSVFVGVFVAVKBVQVFVBGUNVKBHVBGUKVBBULUORSUPUQVCVAGBURUS $.
 
-  $( The image by a Cartesian product.  (Contributed by Thierry Arnoux,
+  $( Image by a Cartesian product.  (Contributed by Thierry Arnoux,
```
Comment (avekens) [inline set.mm:55175]: Maybe "Empty image by ..."

File: set.mm [hunk 5]
```diff
@@ -56389,10 +56395,10 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
 
   ${
     $d x y A $.  $d x y B $.  $d x y R $.
-    $( All nonempty (possibly proper) subclasses of ` A ` , which has a
-       well-founded relation ` R ` , have ` R `-minimal elements.  Proposition
-       6.26 of [TakeutiZaring] p. 31.  (Contributed by Scott Fenton,
-       29-Jan-2011.)  (Revised by Mario Carneiro, 26-Jun-2015.) $)
+    $( All nonempty subclasses of a class having a well-founded set-like
+       relation ` R ` have ` R `-minimal elements.  Proposition 6.26 of
```
Comment (avekens) [inline set.mm:56399]: "well-founded" -> "well-ordered"

File: set.mm [hunk 6]
```diff
@@ -56406,10 +56412,10 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
     $d A y $.  $d B y $.  $d R y $.
     tz6.26i.1 $e |- R We A $.
     tz6.26i.2 $e |- R Se A $.
-    $( All nonempty (possibly proper) subclasses of ` A ` , which has a
-       well-founded relation ` R ` , have ` R `-minimal elements.  Proposition
-       6.26 of [TakeutiZaring] p. 31.  (Contributed by Scott Fenton,
-       14-Apr-2011.)  (Revised by Mario Carneiro, 26-Jun-2015.) $)
+    $( All nonempty subclasses of a class having a well-founded set-like
```
Comment (avekens) [inline set.mm:56415]: "well-founded" -> "well-ordered"

**Review by avekens — CHANGES_REQUESTED** [review]
I think some comments need a revision (clarification or correction), see single review remarks.

File: set.mm (hunk 3 above)
Comment (benjub) [reply set.mm:55114]: I added an explanation in the comment of df-res, and I'll add a link when I add the theorem
```
cnvrescnv $p |- `' ( `' R |` B ) = ( R i^i ( _V X. B ) ) $= ? $.
```

**Comment by sctfn** [comment]
Well-ordered is used for the We predicate which also mandates a total order. Well-founded is used in the literature for our Fr predicate. On Dec 28, 2023, at 12:12 PM, avekens ***@***.***> wrote:﻿
@avekens commented on this pull request.



In set.mm:
> @@ -50906,10 +50904,10 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
       UTVAVDUSVFUTVAVDSSUSVFMUTVAVDABCDFEUDUFUGUHUIURUSUTVEVAURUTMDETZVEURCETUT
       VGCEUJDCEUKULABDEUMUNUOVCADUQUP $.
 
-    $( All nonempty (possibly proper) subclasses of ` A ` , which has a
-       well-founded relation ` R ` , have ` R `-minimal elements.  Proposition
-       6.26 of [TakeutiZaring] p. 31.  (Contributed by Scott Fenton,
-       29-Jan-2011.)  (Revised by Mario Carneiro, 24-Jun-2015.) $)
+    $( All nonempty subclasses of ` A ` , which has a well-founded relation
+       ` R ` , have ` R `-minimal elements.  Proposition 6.26 of

"well-ordered and set-like" instead of "well-founded"?

File: set.mm (hunk 3 above)
Comment (benjub) [reply set.mm:55114]: (done)

File: set.mm [hunk 7]
```diff
@@ -50552,9 +50552,12 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
     $d x y z R $.  $d x y z A $.
     $( Define the well-founded relation predicate.  Definition 6.24(1) of
        [TakeutiZaring] p. 30.  For alternate definitions, see ~ dffr2 and
-       ~ dffr3 .  (Contributed by NM, 3-Apr-1994.) $)
+       ~ dffr3 .  A class is called well-founded when the membership relation
+       ( ~ df-eprel ) is well-founded on it (some sources request that the
+       membership relation be well-founded on its transitive closure).
```
Comment (avekens) [inline set.mm:50557]: Maybe "A class is called well-founded when the membership relation ` _E ` (see ~ df-eprel ) is well-founded on it, i.e., ` _E Fr A ` holds for a well-founded class `A` (some sources request that the membership relation be well-founded on its transitive closure)."

**Review by avekens — COMMENTED** [review]
Only one review remark (on comment for ~ df-fr ) left open.

File: set.mm (hunk 7 above)
Comment (benjub) [reply set.mm:50557]: Done.

**Review by avekens — APPROVED** [review]

**Pull request merged**